recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc2.tar.gz
+  sha256: 254eb82717ffec8b79618c281fe5c87734b2f8d804ef369d414d91b1b71cd163
+
+build:
+  number: 0
+  skip: true  # [win32]
+  msvc_compiler: 14.0
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - toolchain
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib 1.2.*
+    - libmatio >=1.5.12
+    - clangdev 4.0*  # [unix]
+    - libcxx 4.0.*  # [osx]
+    - openmp 4.0.*  # [osx]
+    - libgfortran  # [unix]
+    - openblas 0.2.20|0.2.20.*
+
+  run:
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib 1.2.*
+    - libmatio >=1.5.12
+    - clangdev 4.0*  # [osx]
```

Comment:
In build this is selected for unix and here for osx. Is it on purpose?

Reply:
clang was used for osx, I'll try to use it both in osx and unix, and if it gives troubles I'll set it up only for mac.

Reply:
Normally we use system compilers (though OpenMP is a bit of an exception). We use GCC on Linux, Clang on macOS, and VS on Windows. OpenMP support exists through system compilers (with the exception of macOS).

Using `openmp` and `clangdev` is one way to solve this issue on macOS. The other is using an older `gcc` package.

As we are in the process of migrating compilers, it's difficult to make a good recommendation here. Probably using `clangdev`/`openmp` packages on macOS and `gcc` on Linux will work best as we migrate.

Though asked another conda-forge member who has been working with the compiler migration work to get his take on this. Let's see what he says.